docs/docs.get.md

```diff
@@ -13,7 +15,9 @@ async function myFunc(){
   const goot = new Gootenberg();
   await goot.auth.jwt(credentials);
 
-  await goot.docs.get('MY_DOC_ID');
+  await goot.docs.get('MY_DOC_ID', {
+    suggestionsViewMode: 'PREVIEW_WITHOUT_SUGGESTIONS'
+  });
```

Comment:
I'd rather keep the base example clean. Just like how we handle the extra options in [appendRows](https://github.com/The-Politico/gootenberg/blob/main/docs/sheets.appendRows.md). Just so folks don't think they need to supply this value.

```suggestion
  await goot.docs.get('MY_DOC_ID');
```